src/addonmanager.cpp

```diff
@@ -182,8 +182,6 @@ bool AddonManager::validateIndex(const QByteArray& index,
   for (const AddonData& addonData : addons) {
     if (!m_addons.contains(addonData.m_addonId) &&
         validateAndLoad(addonData.m_addonId, addonData.m_sha256)) {
-      Q_ASSERT(m_addons.contains(addonData.m_addonId));
```

Comment:
This feels unrelated, or has something changed with wasm and the addons?

Reply:
you are right. It's unrelated but with a debug build, the wasm client crashes when validating a blocked addon